### ISSUE TITLE
What dataset did you use for supervised fine-tuning (SFT), and how was it augmented?

### ISSUE THREAD

**Issue by nbasyl** [czg1225/VeriThinker#2]
Hi, thank you for the great work. I’m curious about the experimental details of your SFT setup. Specifically, could you share which dataset was used for the SFT results reported in Table 1, and how the dataset was compressed?

**Comment by nbasyl** [comment]
Hey @czg1225, thanks for getting back to me — but just to clarify, I wasn’t referring to SVFT. I meant which dataset you used for the SFT.

**Comment by czg1225** [comment]
We use the Mixchain-Z-GSM8K and Mixchain-Z-PRM12K Datasets for SFT.

**Comment by nbasyl** [comment]
Thanks a lot, do you simply combine those two dataset as a whole? Also, if you don't mind, would it be possible for you to share the code used to reproduce the SFT results as well?

**Comment by czg1225** [comment]
We simply combine those two datasets and finetune the LRM to a similar compression ratio to compare the accuracy. The SFT code is very similar to our training code.

**Comment by nbasyl** [comment]
There are four types of solutions — which one did you choose for running the SFT?
